### PR TITLE
Accessibility: revision grid branch colors

### PR DESCRIPTION
**Pull request by gerhardol**
Part of #9111 for 3.5
(no PR for master yet, no theme handling even if this commit will not change themes)

This is based of what @ivangrek investigated, I take no credit . Even if this is not final, something should probably be done for 3.5.1 that hopefully can be released soon.
Note especially that no extra investigations for selected commits was done. There are also reports for tag labels.
Note: The tuning was adjusting red to green only, not branch colors to the background which is what matters here.

Edit: Also changed to revert colors to same as in 3.4 (Branch/RemoteBranch still switched compared to 3.4).

## Proposed changes

Adjust colors for branches in the default theme

## Screenshots 

release/3.5 | first tuning | reverted
--|--|--
image | image | image

## Test methodology 

manual

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: GitUI/Themes/dark.css [hunk 1]
```diff
@@ -38,9 +38,9 @@
 .HighlightAllOccurences      { color: #606020; }
 .Tag                         { color: #408080; }
 .Graph                       { color: #882d00; }
-.Branch                      { color: #00d76b; }
+.Branch                      { color: #3d9967; }
 .Branch.colorblind           { color: #0080ff; } /* hsl(210, 100%, 50%) */
-.RemoteBranch                { color: #ff8080; } /* hsl(0,   100%, 75%) */
+.RemoteBranch                { color: #de6459; } /* hsl(0,   100%, 75%) */
```
Comment (mstv) [inline GitUI/Themes/dark.css:43]:
```suggestion
.RemoteBranch                { color: #de6459; }
```

File: GitUI/Themes/darksilver.css [hunk 2]
```diff
@@ -38,12 +38,12 @@
 .HighlightAllOccurences  { color: #606020; }
 .Tag                     { color: #408080; }
 .Graph                   { color: #882d00; }
-.RemoteBranch            { color: #ff8080; } /* hsl(0,   100%, 75%) */
+.RemoteBranch            { color: #de6459; } /* hsl(0,   100%, 75%) */
```
Comment (mstv) [inline GitUI/Themes/darksilver.css:41]:
```suggestion
.RemoteBranch            { color: #de6459; }
```

**Review by mstv — COMMENTED** [review]
Have not run myself.
+1, only if you have swapped the "before" and "after" screenshots. 😉

**Comment by gerhardol** [comment]
> +1, only if you have swapped the "before" and "after" screenshots. 😉

The order was correct, but not the changes that should be done to the invariant theme. Hard to see the changes though, I cannot say that this is better...
Also, the Current branch point is missing for unknown reason

**Comment by RussKie** [comment]
> ## Screenshots
> before	after
> image	image

Sorry, what's the difference besides lost triangles? Why are the triangles are gone?

**Comment by gerhardol** [comment]
> Sorry, what's the difference besides lost triangles? Why are the triangles are gone?

The colors are subtly changed to improve contrast (atleast with standard background).
Triangles lost for unknown reason.

This is a state of how far the issue is resolved. So it is not just to apply the investigation so far...
I am not good at designing and choosing colors and will not take this further...

**Comment by mstv** [comment]
> Triangles lost for unknown reason.

🤔 Might the drawing be denied by `ThreadHelper.AssertOnUIThread()` in `DrawArrow` sometimes? But pretty unlikely. Just came across this while reviewing another PR. I have never noticed missing arrows.

**Comment by gerhardol** [comment]
Updated proposal, new screenshots and arrow visible (using correct alpha).

The original colors look better in my opinion.

@jfrnl also wants to switch back the colors. I was voted down in the original change (that was merged to the 3.5 branch too late in the release process...), I expect that to stay.

**Comment by RussKie** [comment]
> @jfrnl also wants to switch back the colors. I was voted down in the original change (that was merged to the 3.5 branch too late in the release process...), I expect that to stay.

I appreciate it may cause discomfort to some users, but I am of an opinion we made the right call to align with git's colour scheme.

**Review by RussKie — APPROVED** [review]

**Comment by gerhardol** [comment]
Is this better? Should we merge it?

**Comment by RussKie** [comment]
👍from me

**Comment by RussKie** [comment]
Let's merge this

**Comment by nolanhergert** [comment]
Before my "origin/master" branch was green, and now it is red. Also, local branches that were red are now green. Was this intended? Quite confusing when I'm trying to ramp up new people, and I would argue unintuitive since I want to encourage changes to be synced to remote.

**Comment by gerhardol** [comment]
> Before my "origin/master" branch was green, and now it is red. Also, local branches that were red are now green. Was this intended? Quite confusing when I'm trying to ramp up new people, and I would argue unintuitive since I want to encourage changes to be synced to remote.
> 
> EDIT: I less want to complain but more confirm that this is what is intended. I now see the code change is very clearly doing this (as opposed to the initial screenshots above), so it's not accidental. I disagree with the decision, but it's not my project.

Yes, this was intended, changed to use the same color scheme as git-log command line.
This was much debate about this change both before and after the release and you are not the first to ask...

**Pull request merged**